packages/generate-design-tokens/css-generator.js

```diff
@@ -0,0 +1,364 @@
+            return `var(${buildPaletteColorVarName(paletteColorName)})`;
+        }
+
+        // https://regexper.com/#%2Frgba%5C%28%5C%7Bpalette%5C.%28.%2B%29%5C%7D%2C%20%28%5B01%5D%28%3F%3A%5C.%5Cd%2B%29%3F%29%5C%29%2F
+        const colorWithAlphaRegExp = /rgba\(\{palette\.(.+)\}, ([01](?:\.\d+)?)\)/;
+        const colorWithAlphaMatches = colorDescription.value.match(colorWithAlphaRegExp);
+
+        if (colorWithAlphaMatches) {
+            const colorName = colorWithAlphaMatches[1];
+            const alpha = colorWithAlphaMatches[2];
+            usedRawPaletteColors.add(colorName);
+            return `rgba(var(${buildRawPaletteColorVarName(colorName)}), ${alpha})`;
+        }
+
+        throw new Error(`Unknown color format: ${colorDescription.value}`);
+    };
+
+    const lightColors = Object.entries(designTokens.light)
+        .map(
+            ([colorName, colorDescription]) =>
+                `${buildColorVarName(colorName)}: ${buildCssColor(colorDescription)};`
+        )
+        .join('\n');
+
+    const darkModeColors = Object.entries(designTokens.dark)
+        .map(
+            ([colorName, colorDescription]) =>
+                `${buildColorVarName(colorName)}: ${buildCssColor(colorDescription)};`
+        )
+        .join('\n');
+
+    const forceLightModeColors = Object.keys(designTokens.dark)
+        .map(
+            (colorName) => `${buildColorVarName(colorName)}: ${buildCssColor(designTokens.light[colorName])};`
+        )
+        .join('\n');
+
+    const paletteColors = Array.from(usedPaleteColors)
+        .map((colorName) => {
+            return `${buildPaletteColorVarName(colorName)}: ${designTokens.global.palette[colorName].value};`;
+        })
+        .join('\n');
+
+    const rawPaletteColors = Array.from(usedRawPaletteColors)
+        .map((colorName) => {
+            const colorValue = designTokens.global.palette[colorName].value;
+            const [r, g, b] = fromHexToRgb(colorValue);
+            return `${buildRawPaletteColorVarName(colorName)}: ${r}, ${g}, ${b};`;
+        })
+        .join('\n');
+
+    const borderRadii = Object.entries(designTokens.radius)
+        .map(
+            ([radiusName, radiusDescription]) =>
+                `${buildBorderRadiusVarName(radiusName)}: ${buildRadius(radiusDescription)};`
+        )
+        .join('\n');
+
+    const textPresets = {
+        1: {
+            size: {mobile: 12, desktop: 14},
+            lineHeight: {mobile: 16, desktop: 20},
+        },
+        2: {
+            size: {mobile: 14, desktop: 16},
+            lineHeight: {mobile: 20, desktop: 24},
+        },
+        3: {
+            size: {mobile: 18, desktop: 20},
+            lineHeight: {mobile: 24, desktop: 28},
+        },
+        4: {
+            size: {mobile: 20, desktop: 28},
+            lineHeight: {mobile: 24, desktop: 32},
+        },
+        5: {
+            size: {mobile: 20, desktop: 28},
+            lineHeight: {mobile: 24, desktop: 32},
```

Comment:
I think there is a typo in here. Text4 and Text5 have same values. Copy paste error?

Reply:
Yes, and there is also a typo in [text story](https://mistica-web.vercel.app/?path=/story/components-text--text-components)

![image](https://github.com/Telefonica/mistica-web/assets/6722153/86bb7dc4-9b74-41b6-bc60-5f6bd2a54517)

Text2 medium should show Text2 Medium 14/20 (Mobile) | 16/24 (Desktop)


according to specs
https://www.figma.com/design/aNHyvXe1mqjaelE8XWVQD5/%F0%9F%94%B9-Text-presets-definition?node-id=0-1